internal/tasks/schedulertask_test.go

```diff
@@ -140,3 +142,122 @@ func TestSchedulerTask_Run(t *testing.T) {
 	cancel()
 	<-doneCh
 }
+
+func TestSchedulerTask_refreshCache(t *testing.T) {
```

Comment:
this is to increase existing test coverage..?

Reply:
yes just to increase the coverage